cruise-control-client/setup.py

```diff
@@ -0,0 +1,16 @@
+import setuptools
+setuptools.setup(
+    name='cruise-control-client',
+    version='0.0.1',
+    author='mgrubent',
+    author_email='mgrubentrejo@linkedin.com',
+    description='A Python client for cruise-control',
+    url='https://github.com/linkedin/cruise-control',
+    packages=setuptools.find_packages(),
+    install_requires=[
+        'pandas',
+        'requests'
+    ],
+    license='BSD 2-Clause "Simplified"',
```

Comment:
Can we use the same license statement as the rest of the project here?

```
Copyright 2019 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
```

Reply:
Agreed; changed